index.html

```diff
@@ -367,6 +368,29 @@ <h2>
             </ol>
           </dd>
         </dl>
+        <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices">
+          <dt>
+            <dfn>preferredLabel</dfn>
+          </dt>
+          <dd>
+            <p>This settable attribute allows an application to inform the user
+            agent of a meaningful {{MediaStreamTrack/label}} to apply to the
+            tracks that would result if the tab in which the application is 
+            running is captured.<p>
+            <p>Only the top-level document may set or read this value. Attempts
+            to read or write the value from other contexts should result in
+            a {{NotAllowedError}}.
+            </p>
+            <p>If the value is empty (unset, set <code>undefined</code> or set 
+            to the empty string), the application leaves the choice of a
+            label up to the user agent.</p>
+            <p>If set to any non-empty value, the user agent SHOULD incorporate
```

Comment:
UA should also be able to sanitize it in other ways (e..g, if the label is too long)